src/ui-kit/Button/Button.tsx

```diff
@@ -1,13 +1,40 @@
 import { JSX } from '@redneckz/uni-jsx';
+import { useCallback } from '@redneckz/uni-jsx/lib/hooks';
 import { useLink } from '../../hooks/useLink';
+import { type PreventableEvent } from '../PreventableEvent';
 import { ButtonInner } from './ButtonInner';
 import { type ButtonProps } from './ButtonProps';
 import { getDisabledButtonClasses } from './getDisabledButtonClasses';
 import { getRegularButtonClasses } from './getRegularButtonClasses';
 
-export const Button = JSX<ButtonProps>(({ disabled, children, ...rest }) => {
+export const Button = JSX<ButtonProps>(({ disabled, children, method = 'LINK', href, ...rest }) => {
+  const handleFormSubmit = useCallback((ev: PreventableEvent) => {
+    if (method !== 'POST' && href) {
+      return;
+    }
+    ev.preventDefault();
+
+    const url = new URL(href || '');
+    const form = document.createElement('form');
+
+    form.setAttribute('method', 'POST');
+    form.setAttribute('action', url.protocol + url.host + url.pathname);
```

Comment:
либо интерполяция строки (так у нас чаще встречается), либо
```suggestion
    form.setAttribute('action', [origin, pathname].join(''));
```